### PR TITLE
Week2 static elements

### PR DESCRIPTION
**Pull request by jackdavidweber**
Implemented style changes per Danish21's recommendation. Finalized all of the static changes necessary so that I can now focus on doing cool things with javascript like using [A-Frame](https://aframe.io/) and [particle.js](https://vincentgarreau.com/particles.js/)

Here are some mobile screenshots (as requested by augerm)
image
image

And here is a desktop screenshot
image

### CODE REVIEW

File: portfolio/src/main/webapp/template.html [hunk 1]
```diff
@@ -65,56 +82,88 @@ <h2 class="heading">About Me</h2>
...
                                 </div>
-                                
-                                <h3 class="title mb-3"><a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a></h3>
-                                    
+                                <h3 class="title mb-3">
+                                    <a href="#" target="_blank">GradTracks - The best way to visualize alumni paths and outcomes</a>
+                                </h3>                                    
                                 <div class="desc text-left">                                    
-                                    <p>Lead developer for databases class group project. GradTracks is a tool that allows students to see what previous alumni have done after graduating. Built database, backend and frontend that quickly processed queries and displayed data visualizations.</p>
+                                    <p>Lead developer for databases class group project. GradTracks is a tool that allows students to
```
Comment (g-harel) [inline portfolio/src/main/webapp/template.html]: Nit: I like putting text on the next line + indent one level in these situations.

File: portfolio/src/main/webapp/template.html [hunk 2]
```diff
@@ -65,56 +82,88 @@ <h2 class="heading">About Me</h2>
...
+                                    <img 
+                                        class="img-fluid project-image rounded shadow-sm" 
+                                        src="assets/images/projects/microcontroller.jpg" 
+                                        alt="Microcontroller" />
                                 </a>
                                 <div class="desc col-md-8 col-12">
-                                    <h3 class="title"><a href="#" target="_blank">Multicycle Processor</a></h3>
-                                    <p class="mb-2">For the final project of Digital Electronics and Computer Engineering at Harvey Mudd, I built a multicycle RISC-V processor on an FPGA. Involved multiple finite state machines and understanding of microcontroller design from logic gates to the ARM instruction set.  </p>
+                                    <h3 class="title">
+                                        <a href="#" target="_blank">Multicycle Processor</a>
```
Comment (g-harel) [inline portfolio/src/main/webapp/template.html:144]: Are these `href="#"` links part of the template?

File: portfolio/src/main/webapp/template.html (hunk 1 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/template.html]: @g-harel can you clarify? Do you mean put "GradTracks - The best way to visualize alumni paths and outcomes" on its own line indented?

File: portfolio/src/main/webapp/template.html (hunk 2 above)
Comment (jackdavidweber) [reply portfolio/src/main/webapp/template.html:144]: Yes- I am still considering whether I want to remove or link to something

File: portfolio/src/main/webapp/template.html (hunk 1 above)
Comment (g-harel) [reply portfolio/src/main/webapp/template.html]: Yeah sorry, should have done a diff to start. This is how I like to format multiline/long text content:

```diff
- <p>lorem ipsum
+ <p>
+    lorem ipsum
     lorem ipsum
  </p>
```

Comment (jackdavidweber) [reply portfolio/src/main/webapp/template.html]: Thank you @g-harel  - just implemented!

File: portfolio/src/main/webapp/template.html [hunk 3]
```diff
@@ -34,16 +34,30 @@
     <!-- ******HEADER****** --> 
     <header class="header">
         <div class="container clearfix">                       
-            <img class="profile-image img-fluid float-left rounded-circle" style="width:200px;height:200px;" src="assets/images/headshot.jpg" alt="profile image" />
+            <img
+                class="profile-image img-fluid float-left rounded-circle" 
+                style="width:200px;height:200px;" src="assets/images/headshot.jpg"
```
Comment (g-harel) [inline portfolio/src/main/webapp/template.html:39]: Nit: inline styles are hard to override, could probably use existing class and the existing css file.

**Review by g-harel — APPROVED** [review]

**Pull request merged**